fix some QA Issues
http://bugs.gentoo.org/show_bug.cgi?id=149798
Submitted by Robert Buchholz <rob@thetruth.de>

Joerg Bornkessel hd_brummy@gentoo.org 02 Oct 2006
diff -Nuar graphlcd-base-0.1.3.orig/tools/convpic/Makefile graphlcd-base-0.1.3/tools/convpic/Makefile
--- graphlcd-base-0.1.3.orig/tools/convpic/Makefile	2005-11-06 15:17:44.000000000 +0100
+++ graphlcd-base-0.1.3/tools/convpic/Makefile	2006-10-01 21:14:12.000000000 +0200
@@ -32,7 +32,7 @@
 
 install: $(PRGNAME)
 	install -d $(BINDIR)
-	install -m 755 -o root -g root -s $(PRGNAME) $(BINDIR)
+	install -m 755 -o root -g root $(PRGNAME) $(BINDIR)
 
 uninstall:
 	rm -f $(BINDIR)/$(PRGNAME)
diff -Nuar graphlcd-base-0.1.3.orig/tools/crtfont/Makefile graphlcd-base-0.1.3/tools/crtfont/Makefile
--- graphlcd-base-0.1.3.orig/tools/crtfont/Makefile	2005-11-06 15:18:08.000000000 +0100
+++ graphlcd-base-0.1.3/tools/crtfont/Makefile	2006-10-01 21:14:15.000000000 +0200
@@ -35,7 +35,7 @@
 
 install: $(PRGNAME)
 	install -d $(BINDIR)
-	install -m 755 -o root -g root -s $(PRGNAME) $(BINDIR)
+	install -m 755 -o root -g root $(PRGNAME) $(BINDIR)
 
 uninstall:
 	rm -f $(BINDIR)/$(PRGNAME)
diff -Nuar graphlcd-base-0.1.3.orig/tools/genfont/Makefile graphlcd-base-0.1.3/tools/genfont/Makefile
--- graphlcd-base-0.1.3.orig/tools/genfont/Makefile	2005-11-06 15:18:46.000000000 +0100
+++ graphlcd-base-0.1.3/tools/genfont/Makefile	2006-10-01 21:16:34.000000000 +0200
@@ -38,7 +38,7 @@
 
 install: $(PRGNAME)
 	install -d $(BINDIR)
-	install -m 755 -o root -g root -s $(PRGNAME) $(BINDIR)
+	install -m 755 -o root -g root $(PRGNAME) $(BINDIR)
 	
 uninstall:
 	rm -f $(BINDIR)/$(PRGNAME)
diff -Nuar graphlcd-base-0.1.3.orig/tools/showpic/Makefile graphlcd-base-0.1.3/tools/showpic/Makefile
--- graphlcd-base-0.1.3.orig/tools/showpic/Makefile	2005-11-06 15:19:09.000000000 +0100
+++ graphlcd-base-0.1.3/tools/showpic/Makefile	2006-10-01 21:16:39.000000000 +0200
@@ -36,7 +36,7 @@
 
 install: $(PRGNAME)
 	install -d $(BINDIR)
-	install -m 755 -o root -g root -s $(PRGNAME) $(BINDIR)
+	install -m 755 -o root -g root $(PRGNAME) $(BINDIR)
 
 uninstall:
 	rm -f $(BINDIR)/$(PRGNAME)
diff -Nuar graphlcd-base-0.1.3.orig/tools/showtext/Makefile graphlcd-base-0.1.3/tools/showtext/Makefile
--- graphlcd-base-0.1.3.orig/tools/showtext/Makefile	2005-11-06 15:19:43.000000000 +0100
+++ graphlcd-base-0.1.3/tools/showtext/Makefile	2006-10-01 21:16:41.000000000 +0200
@@ -36,7 +36,7 @@
 
 install: $(PRGNAME)
 	install -d $(BINDIR)
-	install -m 755 -o root -g root -s $(PRGNAME) $(BINDIR)
+	install -m 755 -o root -g root  $(PRGNAME) $(BINDIR)
 
 uninstall:
 	rm -f $(BINDIR)/$(PRGNAME)

